pos_picking_load/static/src/xml/pos_picking_load.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<template>
+
+    <t t-name="LoadPickingButtonWidget">
+        <span class="control-button order-split" t-att-style='widget.is_visible() ? "display: block;" : "display: none;"'>
+            <i class="fa fa-upload"></i>
+            <t t-esc="widget.button_text()"/>
+        </span>
+    </t>
+
+    <t t-name="LoadPickingLine">
+        <tr class="picking-line" t-att-data-picking-id="picking.id">
+            <td><t t-esc="picking.name"/></td>
+            <td><t t-esc="picking.partner_id[1] || ''"/></td>
+            <td><t t-esc="picking.min_date || ''"/></td>
+            <td><t t-esc="picking.origin || ''"/></td>
+        </tr>
+    </t>
+
+    <t t-name="LoadPickingScreenWidget">
+        <div class="pickinglist-screen screen">
+            <div class="screen-content">
+                <section class="top-content">
+                    <span class="button back picking-button">
+                        <i class="fa fa-times"></i>
+                        Cancel
+                    </span>
+                    <span class="button validate picking-button highlight">
+                        <i class="fa fa-arrow-circle-o-right"></i>
+                        Select
+                    </span>
+                    <span class="searchbox">
+                        <input placeholder="Search Picking" />
```

Comment:
The search doesn't seem to be handled in JS and it is not doing anything.

Reply:
Oh yes, my bad. Forgot to port this section.